views/proposals.tmpl

```diff
@@ -108,6 +108,8 @@
                         <th class="text-left">Author</th>
                         <th class="text-left">Proposal Status</th>
                         <th class="text-left">Vote Status</th>
+                        <th class="text-left">Votes Count</th>
+                        <th class="text-left">Approval Percentage</th>
```

Comment:
This column can just be "Approval" since there's a "%" with each value.  In general, it's best to keep the table width as narrow as possible.